.gitignore

```diff
@@ -30,3 +30,4 @@ git-src
 .cproject
 .ccache
 .vscode
+.idea
```

Comment:
Please add this in a separate pull request as this is unrelated to the other changes.

Reply:
ok. done :+1: